samplekotlin/src/main/java/com/example/chattutorial/ChannelActivity.kt

```diff
@@ -49,7 +49,6 @@ class ChannelActivity : AppCompatActivity() {
         messageInputViewModel.bindView(binding.messageInputView, this)
 
         // Step 3 - Let both MessageListHeaderView and MessageInputView know when we open a thread
-        // Note: the observe syntax used here requires Kotlin 1.4
```

Comment:
Removing these as Kotlin 1.4 came out last August, and isn't even the latest major version anymore - we can safely assume that users have it now